cell_type_prediction/README.md

```diff
@@ -0,0 +1,56 @@
+# Generating consensus predictions for single-cell RNA-seq cell types
+
+## Objectives
+This tutorial is designed to provide instructions on running the single-cell RNA-seq cell types prediction workflow in Galaxy.
+
+## Introduction 
+When a new, unlabelled scRNA-seq expression data becomes available, it is rarely annotated with cell types. Rigorous annotation requires domain-specific expertise that is not always available in the laboratories that generate the data. To address this problem, a variety of tools have been developed for computational classification of cell types. These tools employ a variety of statistical and mathematical methods. Naturally, each method has inherent advantages and drawbacks. 
+
+This framework employs a widely-used strategy for data classification - it uses multpile tools to obtain predictions, combines them and generates a consensus output. More specifically, it generates predictions for novel data using multiple libraries of classifiers trained on existing datasets with high-quality annotations. Predictions for each cell are initially filtered based on tool-specific scores, followed by calculating the frequency of each candidate label as well as [semantic similarity](https://en.wikipedia.org/wiki/Semantic_similarity) across predictions. These metrics allow to ascertain the reliability and agreement of predictions. Finally, a table with 3 top candidate labels, corresponding metrics per cell and datasets of origin is produced. 
+
+## What you will need
+* Expression data in the format of 10X Genomics directory with the following files: 
+    * `matrix.mtx` - gene-by-cell matrix holding expression data 
+    * `genes.tsv` - list of genes (corresponding to rows of the matrix)
+    * `barcodes.tsv` - list of cell IDs (matrix columns)
+* Collection of pre-trained classifiers (generated by [this](https://github.com/ebi-gene-expression-group/cell-types-train-control-workflow) workflow) 
+* Tool evaluation table (produced as output of [this](https://github.com/ebi-gene-expression-group/cell-types-eval-control-workflow) workflow; available [here](https://www.ebi.ac.uk/~a_solovyev/prod_testing_data/tool_perf_pvals.tsv))
+* [Cell Ontology](http://www.obofoundry.org/ontology/cl.html) graph in .obo or .xml format (available [here](https://www.ebi.ac.uk/~a_solovyev/prod_testing_data/cl-basic.obo))
+* Exclusions file in `.yaml` format (needed to filer out unlabelled cells and trivial terms; see example [here](https://www.ebi.ac.uk/~a_solovyev/prod_testing_data/exclusions.yaml))
+* SDRF files for training datasets (required to extract cell ontology terms)
```

Comment:
Again, we need to figure a way of this happening more simply.

Reply:
@pinin4fjords, supply sdrf files together with pre-trained classifiers?

Reply:
Yes, on reflection, expecting users to go chasing down lots of individual files before they even start isn't going to fly. Maybe we need to prioritise the simple tool I suggested elsewhere in this PR. Given a single dataset ID it could retrieve everything it needs from our FTP for the classifiers.

It should also be more generic to tsv - people don't know what SDRF is really, and all that's needed is a cell/ ontology term mapping, right?